containers/Connector/Connector.tsx

```diff
@@ -209,6 +209,7 @@ const useConnector = () => {
 	};
 
 	return {
+		network,
```

Comment:
it was added since the value happens to be handy in this context and there are a couple of places where it is used with the connector. it doesn't really matter much to me where this value comes from. still want to remove?

Reply:
yeah I think it's fine to leave it. If you can get all you want from the connector without the need to use recoil then all good